src/main/java/Lubot.java

```diff
@@ -0,0 +1,202 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Lubot {
+	private static ArrayList<Task> tasks = new ArrayList<>();
```

Comment:
nit: Should `tasks` have `final` qualifier?

```suggestion
	private static final ArrayList<Task> tasks = new ArrayList<>();
```